site/docs/transform/impute.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+title: Impute
+permalink: /docs/impute.html
+---
+
+To impute data in Vega-Lite, users can either use use an `impute` property of an [encoding field definition](#encoding) or an `impute` transform inside the [`transform`](#transform) array.
```

Comment:
You may need to explain establish the concept of `key` and `groupby` up here so you can use it to explain under encoding.